tests/commands/config/test_config_sample.py

```diff
@@ -36,3 +39,145 @@ def test_write_json(tmp_path, config_files):
+    #GIVEN a dict and json file
+    ref_dict = json.load(open(config_files['reference'], 'r'))
+
+    json_file = config_files['sample']
+
+    # WHEN passing dict and json file to merge
+    merge_dict = merge_json(ref_dict, json_file)
+
+    # THEN It will merge both the data and return dict
+    assert isinstance(merge_dict, dict) == True
+    assert "samples" in merge_dict
+    assert "references" in merge_dict
+
+
+def test_merge_json_error(config_files):
+    with pytest.raises(OSError) as error:
+        #GIVEN a dict and invalid json file path
+        ref_dict = json.load(open(config_files['reference'], 'r'))
+
+        json_file = 'reference.json'
+
+        #WHEN passing python dict and invalid json path
+        merge_dict = merge_json(ref_dict, json_file)
+
+        #THEN it should throw OSError as FileNotFoundError
+        assert 'FileNotFoundError' in error.value
+
+
+def test_set_panel_bed(config_files):
+    #GIVEN test reference json file and panel bed file
+    ref_json = json.load(open(config_files['reference'], 'r'))
+
+    panel_bed = config_files["panel_bed_file"]
+
+    #WHEN passing args to that function
+    json_out = set_panel_bed(ref_json, panel_bed)
+
+    #THEN It will add two more items into the dict
+    assert "chrom" in json_out['bed']
+
+
+def test_set_panel_bed_error(config_files):
+    with pytest.raises(OSError) as error:
+        #GIVEN test reference json file and invalid panel bed file path
+        ref_json = json.load(open(config_files['reference'], 'r'))
+
+        panel_bed = "panel/panel.bed"
+
+        #WHEN passing args to that function
+        json_out = set_panel_bed(ref_json, panel_bed)
+
+        #THEN It will add two more items into the dict
+        assert "FileIsNotFound" in str(error.value)
+
+
+def test_check_exist(config_files):
+    #GVIEN a file path
+    reference_json = config_files['reference']
+
+    #WHEN passing ref file path
+    check = check_exist(reference_json)
+
+    #THEN It will return the boolean value if the file exists in path
+    assert check == True
+
+
+def test_check_exist_error():
+    with pytest.raises(FileNotFoundError) as error:
+        #GIVEN a invalid file path
+        invalid_file_path = "test.txt"
+
+        #WHEN passing a invalid file path
+        check = check_exist(invalid_file_path)
+
+        #THEN It will raise the error
+        assert "FileIsNotFound" in str(error.value)
+
+
+def test_get_analysis_type():
+    #GIVEN umi flag(boolean value) and normal fq file
+    umi_true = True
+    normal_valid = 'normal.fastq'
+    umi_false = False
+    normal_invalid = ''
+    #WHEN passing values
+    #THEN it will return possible analysis type
+    assert 'paired_umi' == get_analysis_type(normal_valid, umi_true)
+    assert 'single_umi' == get_analysis_type(normal_invalid, umi_true)
+    assert 'paired' == get_analysis_type(normal_valid, umi_false)
+    assert 'single' == get_analysis_type(normal_invalid, umi_false)
```

Comment:
In all four above asserts, swap the comparisons.

```suggestion
    assert get_analysis_type(normal_invalid, umi_false) == 'single'
```